<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа 2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,5">
    <item>
     <layout class="QVBoxLayout" name="LeftLayout" stretch="2,1,0">
      <item>
       <layout class="QVBoxLayout" name="Matrlayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Матрица инцидентности</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="incEdit">
          <property name="font">
           <font>
            <family>Courier New</family>
           </font>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="resultGroup">
        <property name="title">
         <string>Итоги работы</string>
        </property>
        <layout class="QVBoxLayout" name="resultLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Результат</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="resEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Промежуточный результат</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="stackEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ControlLayout" stretch="1,1">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QGroupBox" name="LeftControlGroup">
          <property name="title">
           <string>Редактирование графа</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QLineEdit" name="NodeNameEdit">
             <property name="placeholderText">
              <string>Введите имя элемента</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="AddDelLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetNoConstraint</enum>
             </property>
             <item>
              <widget class="QPushButton" name="addNodeButton">
               <property name="text">
                <string>Добавить</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteNodeButton">
               <property name="text">
                <string>Удалить</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="desButton">
             <property name="text">
              <string>Дезориентировать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="AddEdge">
             <property name="text">
              <string>Добавить связь</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="DeleteEdgeButton">
             <property name="text">
              <string>Удалить связь</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ClearButton">
             <property name="text">
              <string>Очистить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="matrButton">
             <property name="text">
              <string>Построить матрицу</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="RightContolLayoyt" stretch="0,0">
          <item>
           <widget class="QGroupBox" name="ImageControlGroup">
            <property name="title">
             <string>Отображение</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>Показать веса</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="gravityBox">
               <property name="text">
                <string>Гравитация</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="colorButton">
               <property name="text">
                <string>Случайные цвета</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="shuffleButton">
               <property name="text">
                <string>Перетасовать</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="AlgContolBox">
            <property name="title">
             <string>Алгоритм</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Пуск</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clrMarksButton">
               <property name="text">
                <string>Сброс</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Скорость</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="speedSlider">
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="stepBox">
               <property name="text">
                <string>Пошагово</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="graphLayout" stretch="0,0">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Граф</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="open_action"/>
    <addaction name="save_action"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="open_action">
   <property name="text">
    <string>Открыть граф</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>Сохранить граф</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
